ReactiveCocoa/Swift/Action.swift

```diff
@@ -191,13 +191,7 @@ public final class CocoaAction: NSObject {
 	/// object given to execute(), if it can be downcast successfully, or nil
 	/// otherwise.
 	public convenience init<Input: AnyObject, Output, Error>(_ action: Action<Input?, Output, Error>) {
-		self.init(action, { input in
-			if let input: AnyObject = input {
-				return input as? Input
-			} else {
-				return nil
-			}
-		})
+		self.init(action, { $0 as? Input })
```

Comment:
When `$0 as? Input?` is used, the return type is `Input??`. To specify `Input?` as the casting type, we should use `as!` operator: `$0 as! Input?`

I think `$0 as? Input` is safer and more understandable than `$0 as! Input?`.